go/store/val/offsets.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package val
+
+import "encoding/binary"
+
+type Offsets []byte
```

Comment:
Do any of the Offsets methods need to be exported?  Should they all be internal?

Reply:
In the scope of this PR they're all internal, but I reused this type for the offset array in prolly tree nodes